src/server/views/dashboard/partials/list-item-summary.njk

```diff
@@ -20,23 +20,21 @@
       {{ listItem.contactName }}
     </h2>
   </div>
-  <div class="dashboard__list-item-tag-container">
-      {% include "./list-item-tag.njk" %}
-  </div>
   <div class="dashboard__list-item-body">
+    <div>
+      {% include "./list-item-tag.njk" %}
+      <div class="dashboard__list-item-tag-container">
+        {% include "./list-item-activity-status-tag.njk" %}
+      </div>
+    </div>
```

Comment:
did this merge apply correctly?

Reply:
Looks OK as far as I can see. Labels appear on left and right hand side of the row as expected.

Reply:
these UI changes should already be in master - i.e. no change, assumedly?